### PR TITLE
draw centered text

### PR DESCRIPTION
**Pull request by badjavadev**
## Description

i need this i need this i need this i need this i need this i need this

## Checklist
- [x] I made a clear description of what was changed
- [x] I stated why these changes were necessary
- [x] I updated documentation or said what needs to be updated
- [x] I made sure these changes are backwards compatible
- [x] This pull request is for one feature/bug fix

### CODE REVIEW

**Comment by badjavadev** [comment]
yes this is useless

File: src/main/java/cc/polyfrost/oneconfig/gui/elements/config/ConfigKeyBind.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ public void draw(long vg, int x, int y, InputHandler inputHandler) {
                 OneConfigGui.INSTANCE.allowClose = true;
                 inputHandler.stopBlockingInput();
             }
-        } else if (text.equals("")) text = "None";
```
Comment (Wyvest) [inline src/main/java/cc/polyfrost/oneconfig/gui/elements/config/ConfigKeyBind.java:97]: what

**Review by Wyvest — CHANGES_REQUESTED** [review]
apart from that one thing it looks good

File: src/main/java/cc/polyfrost/oneconfig/gui/elements/config/ConfigKeyBind.java (hunk 1 above)
Comment (badjavadev) [reply src/main/java/cc/polyfrost/oneconfig/gui/elements/config/ConfigKeyBind.java:97]: idk what is this

File: src/main/java/cc/polyfrost/oneconfig/renderer/NanoVGHelper.java [hunk 2]
```diff
@@ -177,6 +177,20 @@ public interface NanoVGHelper {
      */
     void drawText(long vg, String text, float x, float y, int color, float size, Font font);
 
+    /**
+     * Draws a Centered String with the given parameters.
```
Comment (TheKodeToad) [inline src/main/java/cc/polyfrost/oneconfig/renderer/NanoVGHelper.java:181]:
```suggestion
     * Draws a centered String with the given parameters.
```

**Pull request closed without merging**